distributedlog-service/bin/dlog-daemon.sh

```diff
@@ -43,7 +43,7 @@ fi
 
 # DLOG logging configuration
 DLOG_LOG_DIR=${DLOG_LOG_DIR:-"$DL_HOME/logs"}
-DLOG_ROOT_LOGGER=${DLOG_ROOT_LOGGER:-'INFO,ROLLINGFILE'}
+DLOG_ROOT_LOGGER=${DLOG_ROOT_LOGGER:-'INFO,R'}
```

Comment:
FileAppender `ROLLINGFILE` does not exist in default log4j.properties, it's `R` instead.